.github/workflows/build.yml

```diff
@@ -52,10 +52,10 @@ jobs:
     strategy:
       matrix:
         include:
-        - python-version: 3.8
-          numpy-version: 1.21
         - python-version: '3.10'
-          numpy-version: 1.24
+          numpy-version: 1.23
+        - python-version: 3.12
+          numpy-version: 1.26
```

Comment:
Is this testing the minimum and the max version combos?

Reply:
Yup